packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -94,7 +94,7 @@ export const participantListSelector: ParticipantListSelector = createSelector(
       : [];
     participants.push({
       userId: userId,
-      displayName: displayName,
+      displayName: displayName ?? '',
```

Comment:
Was this needed?

Reply:
Ideally stateful can propagate undefined safely to signify that there was no display name set

Reply:
It is needed so that unnamed participants are actually pushed into the participants array